packages/x-components/src/router.ts

```diff
@@ -9,6 +9,11 @@ const routes: RouteConfig[] = [
     name: 'Layout',
     component: () => import('./views/Layout.vue')
   },
+  {
+    path: '/jumbo',
+    name: 'Jumbo',
```

Comment:
Name it something like Fixed header layout?

Reply:
+1